src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -86,7 +86,7 @@ public RevisionsStorage(DocumentDatabase database, Transaction tx)
                 Default = new RevisionsCollectionConfiguration
```

Comment:
Need to review all creation of such things, to ensure that we don't have `Active` missing and meant to be disabled there.

Reply:
found only one place where we create such things and the flag is missing, and changed it